src/ConfigManagerHelper.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content;
+
+use Drupal\Core\Config\ConfigManagerInterface;
+use Drupal\Core\Config\StorageInterface;
+
+/**
+ * The ConfigManagerHelper class.
+ *
+ * It provides additional methods to work with the configuration system.
+ */
+class ConfigManagerHelper implements ConfigManagerHelperInterface {
+
+  /**
+   * Configuration manager.
+   *
+   * @var \Drupal\Core\Config\ConfigManagerInterface
+   */
+  protected $configManager;
+
+  /**
+   * Entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * ConfigurationManagerHelper constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigManagerInterface $config_manager
+   *   Config manager.
+   */
+  public function __construct(ConfigManagerInterface $config_manager) {
+    $this->configManager = $config_manager;
+    $this->entityTypeManager = $config_manager->getEntityTypeManager();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function createConfig(string $config_name, StorageInterface $storage): void {
+    $entity_type = $this->configManager->getEntityTypeIdByName($config_name);
+
+    $config_values = $storage->read($config_name);
+    if (!$this->entityTypeManager->getStorage($entity_type)->load($config_values['id'])) {
+      $entity_type_definition = $this->entityTypeManager->getDefinition($entity_type);
+      $entity_type_class = $entity_type_definition->getOriginalClass();
+      $entity_type_class::create($config_values)->save();
+    }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function updateConfig(string $config_name, StorageInterface $storage, array $field_names = []): void {
```

Comment:
The idea of a centralized service is not bad per se, only I'm not sure about it's usage in this specific place. We should also consider such a service to become, maybe, a contributed module or library.

Let's remove this service for now, and just update the phone cardinality, let's rather open a follow up where we can discuss if to create a separate service for this.

Reply:
We could use this issue to investigate the service right? https://citnet.tech.ec.europa.eu/CITnet/jira/browse/D8TEWPPE-53